$OpenBSD$

Turn off auto-update stuff.  Turn on data-in-home-dir
--- .mozconfig.orig	Wed Sep  7 02:07:10 2016
+++ .mozconfig	Tue Sep 20 19:57:12 2016
@@ -9,10 +9,11 @@ ac_add_options --enable-optimize
 #ac_add_options --disable-optimize
 ac_add_options --enable-official-branding
 
-ac_add_options --enable-tor-browser-update
-ac_add_options --enable-update-packaging
-ac_add_options --enable-signmar
-ac_add_options --enable-verify-mar
+#ac_add_options --enable-tor-browser-update
+#ac_add_options --enable-update-packaging
+#ac_add_options --enable-signmar
+#ac_add_options --enable-verify-mar
+ac_add_options --enable-tor-browser-data-in-home-dir
 
 ac_add_options --disable-strip
 ac_add_options --disable-install-strip
